deployment/jupyterhub/deploy.sh

```diff
@@ -22,16 +22,11 @@ fi
 VAR_FILE=$(realpath $VAR_FILE)
 source $VAR_FILE
 
+namespace=$1
+
```

Comment:
same comment from above - check if `$1` exists, and error if not

Reply:
done